lib/auth/providers/index.ts

```diff
@@ -65,8 +65,8 @@ export const auth = betterAuth({
   advanced: {
     defaultCookieAttributes: {
       httpOnly: true,
-      secure: COOKIE_SECURE,
-      sameSite: COOKIE_SAME_SITE,
+      secure: true,
```

Comment:
we can remove the `defaultCookieAttributes` object from the config as it was before #307 ?